src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get"
+      data-qa="keypress"
+    >
+      <img
+        class="bigSearchImg"
+        src="images/Search.svg"
+        alt="search"
+      >
+      <input
+        class="bigSearch searching"
```

Comment:
[checklist p.2](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

you can leave just `searching` class there. Also don't forget to move styles

Reply:
I have a question about this, wouldn't it be a problem to set the styles for both elements first and then change the styles again for the small search bar?

Or during display for the browser will be a greater load to add to the elements those styles that are unique for each search bar?

Reply:
no, that would not be a problem. Try and you will see how it works in practice